log/logger.go

```diff
@@ -95,11 +97,15 @@ func (l *Logger) Logf(level logrus.Level, category string, msg string, args ...a
 		fmt.Printf("%s [%d]: %s - %s ms\n", magenta(category), goRoutineID(), string(msg), magenta(elapsed))
 		return
 	}
-	entry := l.WithFields(logrus.Fields{
+	fields := logrus.Fields{
 		"category":  category,
 		"elapsed":   fmt.Sprintf("%d ms", elapsed),
 		"goroutine": goRoutineID(),
-	})
+	}
+	if l.GetLevel() > logrus.InfoLevel {
+		fields["iteration_id"] = l.iterID
+	}
```

Comment:
This will print `iteration_id=` if `l.iterID` is empty. Not a big deal, but should we do the following?

```suggestion
	if l.iterID != "" && l.GetLevel() > logrus.InfoLevel {
		fields["iteration_id"] = l.iterID
	}
```

So we can skip seeing the `iteration_id` in the logs if it's empty.

Reply:
Resolved by a7e79ce.